internal/sdkv2provider/resource_cloudflare_workers_secret_test.go

```diff
@@ -0,0 +1,104 @@
+package sdkv2provider
+
+import (
+	"context"
+	"fmt"
+	"os"
+	"testing"
+
+	"github.com/cloudflare/cloudflare-go"
+	"github.com/hashicorp/terraform-plugin-testing/helper/resource"
+	"github.com/hashicorp/terraform-plugin-testing/terraform"
+)
+
+const scriptContentForSecret = `addEventListener('fetch', event => {event.respondWith(new Response('test 1'))});`
+
+var workerSecretTestScriptName string
+
+func TestAccCloudflareWorkerSecret_Basic(t *testing.T) {
+	t.Parallel()
+
+	name := generateRandomResourceName()
+	secretText := generateRandomResourceName()
+	workerSecretTestScriptName = generateRandomResourceName()
+	accountId := os.Getenv("CLOUDFLARE_ACCOUNT_ID")
+
+	resource.Test(t, resource.TestCase{
+		PreCheck: func() {
+			testAccPreCheck(t)
+			testAccPreCheckAccount(t)
+		},
+		ProviderFactories: providerFactories,
+		CheckDestroy:      testAccCheckCloudflareWorkerSecretDestroy,
+		Steps: []resource.TestStep{
```

Comment:
I've added back a separate import secret test to be consistent with the other tests. This is currently failing with:

```
=== RUN   TestAccCloudflareWorkerSecret_Import
    import_cloudflare_worker_secret_test.go:17: Step 2/2 error running import: exit status 1
        
        Error: invalid id ("0e5935a6b09194ee32e735261911e922") specified, should be in format "accountID/scriptName/secretName"
        
        
--- FAIL: TestAccCloudflareWorkerSecret_Import (23.28s)
```

This is because the Cloudflare API doesn't let you read back the secret text as it's encrypted. I'm not sure how to reconcile this to make the tests pass?

Reply:
Got the tests to pass by ignoring state verification for that field